** Some documentation references to /usr/local/ have been changed to /usr.
   Provided by: Romain Francoise <rfrancoise@debian.org>

--- a/doc/man/emacs.1
+++ b/doc/man/emacs.1
@@ -630,28 +630,28 @@
 .
 .
 .SH FILES
-/usr/local/share/info \(em files for the Info documentation browser.
+/usr/share/info \(em files for the Info documentation browser.
 The complete text of the Emacs reference manual is included in a
 convenient tree structured form.
 Also includes the Emacs Lisp Reference Manual, useful to anyone
 wishing to write programs in the Emacs Lisp extension language.
 
-/usr/local/share/emacs/$VERSION/lisp \(em Lisp source files and
+/usr/share/emacs/$VERSION/lisp \(em Lisp source files and
 compiled files that define most editing commands.
 Some are preloaded; others are autoloaded from this directory when
 used.
 
-/usr/local/libexec/emacs/$VERSION/$ARCH \(em various programs that are
+/usr/lib/emacs/$VERSION/$ARCH \(em various programs that are
 used with GNU Emacs.
 
-/usr/local/share/emacs/$VERSION/etc \(em various files of information.
+/usr/share/emacs/$VERSION/etc \(em various files of information.
 
-/usr/local/share/emacs/$VERSION/etc/DOC.* \(em contains the documentation
+/usr/share/emacs/$VERSION/etc/DOC.* \(em contains the documentation
 strings for the Lisp primitives and preloaded Lisp functions
 of GNU Emacs.
 They are stored here to reduce the size of Emacs proper.
 
-/usr/local/share/emacs/$VERSION/etc/SERVICE lists people offering
+/usr/share/emacs/$VERSION/etc/SERVICE lists people offering
 various services to assist users of GNU Emacs, including education,
 troubleshooting, porting and customization.
 .
@@ -675,7 +675,7 @@
 
 Please do not send anything but bug reports to this mailing list.
 For more information about Emacs mailing lists, see the
-file /usr/local/emacs/etc/MAILINGLISTS.
+file /usr/share/emacs/$VERSION/etc/MAILINGLISTS.
 Bugs tend actually to be fixed if they can be isolated, so it is
 in your interest to report them in such a way that they can be
 easily reproduced.
